### DISCUSSION TITLE
Encountered a small problem while performing a prerequisite task (401-object-detection-webcam notebook).

### DISCUSSION THREAD

**Discussion by AnuragTimilsina** [openvinotoolkit/openvino#16717] [Google Summer of Code]
Hello OpenVino family, 

As I was trying to complete my prerequisite task I encountered an anomaly in the notebook.  At the second last cell, the webcam video was flickering and hence I set ```use_popup=True```. Now, it has been forever, the cell keeps on executing but there's no popup window for inference. What could be the issue?

**Comment by adrianboguszewski** [comment]
Unfortunately, the flickering occurs on Firefox. Please use Chrome to avoid that.

Check if there is no popup window in the background.
